packages/mobile/src/exchange/ExchangeReview.tsx

```diff
@@ -136,8 +227,38 @@ class ExchangeReview extends React.Component<Props> {
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: colors.background,
-    paddingTop: 20,
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+  },
+  paddedContainer: {
+    paddingHorizontal: 16,
+  },
+  column: {
```

Comment:
Nit: strange name for something that just justifies to flex-start

Reply:
Good point. Renamed to flexStart